src/hotrod/test/Unit.cpp

```diff
@@ -23,9 +29,101 @@ volatile int passFail = 0;
 volatile int available = 0;
 volatile bool waitersDone = false;
 
+#define UNUSED(x) (void)(x)
+const int CODEC_TEST_NUMBER_OF_TESTS = 123;
+const int CODEC_TEST_NUMBER_OF_ITERATIONS_PER_TEST = 1023;
+
+using namespace infinispan::hotrod;
+using namespace infinispan::hotrod::protocol;
 using namespace infinispan::hotrod::sys;
 using namespace infinispan::hotrod::transport;
 
+class TestTransport : public AbstractTransport {
+
+public:
+    TestTransport() : AbstractTransport(*(TransportFactory*)NULL) {
+    }
+
+    void flush() {
+
+    }
+
+    void writeByte(uint8_t var) {
+        UNUSED(var);
```

Comment:
You could also simply comment-out the unused parameter "void writeByte(uint8_t /*var*/) {".